tests/functional/reset_password.js

```diff
@@ -151,16 +151,20 @@ define([
         .findById('fxa-reset-password-header')
         .end()
 
-        // ensure there is a back button
-        .findById('back')
+        // ensure there is a signin button
+        .findByCssSelector('a[href="/signin"]')
         .end()
 
-        // email should be pre-filled
+        // ensure there is a link to `Learn how Sync works`.
+        // TODO Specify actual link
+        .findByCssSelector('a[href="#"]')
+        .end()
+
+        // email should not be pre-filled
         .findByCssSelector('input[type="email"]')
```

Comment:
You can use [testElementValueEquals](https://github.com/mozilla/fxa-content-server/blob/f1337b228ae027bdc988165f2b86b1487457e2da/tests/functional/lib/helpers.js#L802) to reduce boilerplate here.